src/releaseBusV2/release-bus-v2.reconciler.ts

```
@@ -2459,6 +2476,10 @@ export class ReleaseBusV2Reconciler {
   ): Promise<PreparedRepository> {
     const train = context.train;
     const candidates = relevantCandidates(context, repository);
+    const evidenceCandidates = candidateEvidenceCandidates(
+      context,
+      repository
+    );
```

ESLint

[error] Replace `⏎······context,⏎······repository⏎····` with `context,·repository`

(prettier/prettier)